build.rs

```diff
@@ -136,14 +150,25 @@ mod generate {
 
         wrapper.flush()?;
 
-        // Then generate bindings from that file
-        bindgen::builder()
+        let mut builder = bindgen::builder()
             .ctypes_prefix("crate::_fake_ctypes")
             .header(wrapper_path.as_os_str().to_str().unwrap())
-            .whitelist_var("DRM_FOURCC_.*")
-            .generate()
-            .unwrap()
-            .write_to_file("src/consts.rs")?;
+            .whitelist_var("DRM_FOURCC_.*");
+
+        // Add additional include paths
+        for include_path in &extra_includes {
+            let path = format!(
+                "{}",
+                include_path
+                    .as_os_str()
+                    .to_str()
+                    .expect("path is not valid utf8")
+            );
```

Comment:
That's quite the mess, you can call `.to_str()` directly on `&PathBuf`.

There's even a clippy lint to tell you that `format!("{}", some_str)` is useless here...

Reply:
Not sure why tihs was marked as resolved, there's still a redundant `as_os_str()` here.